pkg/admission/apibinding/apibinding_admission.go

```diff
@@ -235,3 +315,16 @@ func (o *apiBindingAdmission) ValidateInitialization() error {
 func (o *apiBindingAdmission) SetDeepSARClient(client kcpkubernetesclientset.ClusterInterface) {
 	o.deepSARClient = client
 }
+
+func (o *apiBindingAdmission) SetKcpInformers(informers kcpinformers.SharedInformerFactory) {
+	apiExportsReady := informers.Apis().V1alpha1().APIExports().Informer().HasSynced
+	o.SetReadyFunc(func() bool {
+		return apiExportsReady()
+	})
+	o.apiExportLister = informers.Apis().V1alpha1().APIExports().Lister()
+	o.apiExportIndexer = informers.Apis().V1alpha1().APIExports().Informer().GetIndexer()
+
+	indexers.AddIfNotPresentOrDie(informers.Tenancy().V1alpha1().WorkspaceTypes().Informer().GetIndexer(), cache.Indexers{
```

Comment:
At some point VerySoon™, let's deduplicate index functions. All the generated informers use kcpcache.ClusterIndexFunc and ClusterAndNamespaceIndexFunc.